src/software/networking/BUILD

```diff
@@ -1,5 +1,3 @@
-package(default_visibility = ["//visibility:public"])
-
```

Comment:
I don't think it defaults to private

Reply:
It does https://docs.bazel.build/versions/master/visibility.html#visibility-of-a-rule-target